rules/windows/credential_access_mimikatz_powershell_module.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/12/07"
+maturity = "production"
+updated_date = "2020/12/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Mimikatz is a credential dumper capable of obtaining plaintext Windows account logins and passwords, along with many
+other features that make it useful for testing the security of networks. This rule detects the Invoke-Mimikatz
+PowerShell command.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Mimikatz Powershell Module Activity Detected"
+note = "This rule identifies an adversary attempt to collect, decrypt, and/or use cached credentials. Alerts from this rule should be prioritized because an adversary has an initial foothold onto an endpoint."
+references = ["https://attack.mitre.org/software/S0002/"]
+risk_score = 99
+rule_id = "ac96ceb8-4399-4191-af1d-4feeac1f1f46"
+severity = "critical"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+type = "query"
+
+query = '''
+event.category:process and event.type:start and process.name:(cmd.exe or powershell.exe)
+and process.args:(*DumpCreds or *Mimikatz* or *mimikatz*)
```

Comment:
looks good, some extra cmd args you may want to group in one rule, better use EQL for case insensitivity: 

privilege::debug 
sekurlsa::logonPasswords
lsadump::dcsync
vault::cred
vault::list
kerberos::tgt
kerberos::ptt
token::elevate
pr::dg 
sl::lp
Get-VaultCredential
Get-Key
Get-Keystrokes
Get-GPPAutologon
Get-GPPPassword
Invoke-NinjaCopy
Invoke-TokenManipulation
Out-Minidump